src/components/Config/Config.tsx

```diff
@@ -8,9 +8,28 @@ import { capitalizeFirstLetter } from '../../utils';
 import CopyInput from '../CopyInput';
 import PageTitle from '../PageTitle';
 import ConnectionSettings from './ConnectionSettings';
+import { detect } from 'detect-browser';
```

Comment:
move this import to the external imports section above (all the imports that do not start with `./` or `../`)